### PR TITLE
Aswathy/Translation 909 derived to indices

### PR DESCRIPTION
**Pull request by aswathy-deriv**
Changes:

-   ...

## Type of change

-   [ ] Bug fix
-   [ ] New feature
-   [ ] Update feature
-   [ ] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

**Review by mayuran-deriv — APPROVED** [review]

**Review by meenakshi-deriv — APPROVED** [review]

**Review by hasan-deriv — APPROVED** [review]

**Review by Nuzhy-Deriv — APPROVED** [review]

File: src/pages/trade-types/multiplier/_how-options-works.tsx [hunk 1]
```diff
@@ -127,7 +127,7 @@ const HowOptionsWorks = () => {
                     >
                         {is_eu ? (
                             <StaticImage
-                                src="../../../images/common/trade-types/multiplier-market-eu.png"
```
Comment (habib-deriv) [inline src/pages/trade-types/multiplier/_how-options-works.tsx:130]: Hi @aswathy-deriv , can we delete the old images if it's not used anywhere?

File: src/pages/trade-types/multiplier/_how-options-works.tsx [hunk 2]
```diff
@@ -127,7 +127,7 @@ const HowOptionsWorks = () => {
...
+                                src="../../../images/common/trade-types/multiplier-market-eu-new.png"
```
Comment (habib-deriv) [inline src/pages/trade-types/multiplier/_how-options-works.tsx:130]: Same

File: src/pages/trade-types/multiplier/_how-options-works.tsx [hunk 3]
```diff
@@ -137,7 +137,7 @@ const HowOptionsWorks = () => {
                             />
                         ) : (
                             <StaticImage
-                                src="../../../images/common/trade-types/multiplier-market.png"
+                                src="../../../images/common/trade-types/multiplier-market-new.png"
```
Comment (habib-deriv) [inline src/pages/trade-types/multiplier/_how-options-works.tsx:140]: Same

File: src/pages/trade-types/multiplier/_how-options-works.tsx (hunk 1 above)
Comment (aswathy-deriv) [reply src/pages/trade-types/multiplier/_how-options-works.tsx:130]: okay

File: src/pages/trade-types/multiplier/_how-options-works.tsx (hunk 3 above)
Comment (aswathy-deriv) [reply src/pages/trade-types/multiplier/_how-options-works.tsx:140]: okay

File: src/pages/trade-types/multiplier/_how-options-works.tsx (hunk 2 above)
Comment (aswathy-deriv) [reply src/pages/trade-types/multiplier/_how-options-works.tsx:130]: okay

**Review by habib-deriv — APPROVED** [review]
LGTM

**Review by fasihali-deriv — APPROVED** [review]

**Pull request merged**